python/pywatchman/pybser.py

```diff
@@ -403,12 +411,22 @@ def loads_recursive(self, buf, pos):
                              binascii.hexlify(val_type).decode('ascii'))
 
 def pdu_len(buf):
-    if buf[0:2] != EMPTY_HEADER[0:2]:
+    # The C-extension version of BSER tools does the detection of the protocol
```

Comment:
(This comment isn't necessary -- it's implied that the C and Python versions have the same interface.)